docs/docker-hub/accounts.md

```diff
@@ -79,7 +79,7 @@ repository access rights to groups:
 - `Write` access users are able to push to non-automated repositories on the Docker
   Hub.
 - `Admin` access allows the user to modify the repositories "Description", "Collaborators" rights,
-  "Mark as unlisted", "Public/Private" status and "Delete".
+  "Mark as unlisted", "Public/Private" status and "Delete", and to retrieve build logs.
```

Comment:
No.
"Automated build" repositories trigger a new build of the Docker image whenever a change occurs in the tracked repository. Every build is recorded and visible through the *builds history* page for users with Read permission. But it is required to have Admin permission to actually read a build log.